ntbs-service/wwwroot/source/Components/ValidateMultiple.ts

```diff
@@ -0,0 +1,56 @@
+import Vue from 'vue';
+import { getHeaders, getValidationPath } from '../helpers';
+const axios = require('axios');
+
+const ValidateMultiple = Vue.extend({
+    props: {
+        model: String,
+        properties: Array
+    },
+    data: function() {
+        return {
+            hasError: false
+        }},
+    methods: {
+        // Annotate values to compare with ref="input[i]", and error fields with ref="errorField[i]", for any number of inputs that should be validated together
+        // Each error message will be mapped to the corresponding field.

```

Comment:
I tried for ages making this work in a more vue-like way by binding directly to v-model; however I can't figure out how to bind the initial value to the value from our razor page model, so using refs instead (which unfortunately don't automatically bind to arrays, unlike what you can do with data/props)

Reply:
i don't think we can use `bind` if vue isn't the sole thing responsible for the value, which is our case :( refs it has to be im afraid

Reply:
Yeah I was hoping you can at least initialise vue with dynamic values from other sources but it doesn't look like it